scripts/scenarios/cloud7/cloud7-ironic-basic-ha.yml

```diff
@@ -112,15 +112,6 @@ proposals:
             start: ##ironic_net_prefix##.21
             end: ##ironic_net_prefix##.254
         mtu: 1500
-  deployment:
-    elements:
-      switch_config:
-      - crowbar.virtual.cloud.suse.de
-      network:
-      - crowbar.virtual.cloud.suse.de
-      - "@@controller1@@"
-      - "@@compute@@"
-      - "@@controller2@@"
```

Comment:
This was the default deployment anyway. With this part removed it seems that I get the complete set of nodes as default.